src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -213,14 +212,6 @@ public void sanitizeForSaving() {
         this.courseId = Sanitizer.sanitizeForHtml(courseId);
         this.giverEmail = Sanitizer.sanitizeForHtml(giverEmail);
 
-        if (recipients != null) {
```

Comment:
This means we have comments that are html-sanitized in the db too?Do we need a script to unsanitise these too?